pyproject.toml

```diff
@@ -10,11 +10,13 @@ dependencies = []
 dev = [
     "black>=25.1.0",
     "docformatter>=1.7.5",
+    "pydata-sphinx-theme>=0.16.1",
```

Comment:
Do these belong at the top level here or should these be added into the generated projects dependencies?